app/views/ems_container/_main.html.haml

```diff
@@ -1 +1,13 @@
-= render :partial => "shared/views/ems_common/main"
+= render :partial => "layouts/flash_msg"
+.row
+  .col-sm-12.col-md-12.col-lg-6
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
+                                                               :items => textual_group_properties}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Status"), :items => textual_group_status}
+  .col-sm-12.col-md-12.col-lg-6
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"),
+                                                               :items => textual_group_relationships}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Topology"),
+                                                               :items => textual_group_topology}
+    = render :partial => "shared/summary/textual_tags", :locals => {:title => _("Smart Management"),
+                                                                    :items => textual_group_smart_management}
```

Comment:
Still no newline, I actually checked and haml-lint handles it just fine, what's the problem?